MessengerOnFileForLinux/ChatControl/src/UserInactivityDetector.cpp

```diff
@@ -0,0 +1,29 @@
+#include "UserInactivityDetector.hpp"
```

Comment:
ChatControl to według ciebie dobre miejsce na tą klase?

Reply:
Troche jest rozmyte to sprzątanie. Coś się sprząta w signal hnadlingu, coś w terminal funciotnality.Ja bym to zostawił tutaj. Pasuje pod ChatControl